aptos-move/vm-genesis/src/lib.rs

```diff
@@ -211,7 +208,7 @@ fn exec_function(
                 "Error calling {}.{}: {}",
                 module_name,
                 function_name,
-                e.into_vm_status()
+                e // .into_vm_status()
```

Comment:
I'd delete this comment if it's no longer used.

Reply:
Done